src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,380 @@
+---
+title: Distributed Tracing
+sidebar_order: 1
+---
+
+[Alert Box: What plan APM is available on? ]
+
+Enabling tracing data collection can augment your existing errors to answer any question or unknowns about your systems. Tracing also allows you to track critical metrics and service-level objectives (SLO) like throughput, latency, or apdex to build a complete monitoring solution to diagnose and measure your application health.
+
+## Tracing & Distributed Tracing
+
+Applications (for example, web applications) typically consist of interconnected applications/components (also called services). For instance, let us assume a modern web application could be composed of the following components separated by the network boundaries:
+
+- Frontend (Single-Page Application)
+- Backend
+- Search Service
+- Cache Service
```

Comment:
@dashed Maybe we should have these match the diagram? So, it'll be:

- Frontend
- Backend
- Background Queue
- Notification Job

Reply:
Yes! That's one of the fixes I wanted to suggest.